cvat-ui/src/components/annotation-page/standard3D-workspace/context-image/context-image.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useEffect } from 'react';
+
+interface Props {
+    frame: number;
+    contextImageHide: boolean;
+    loaded: boolean;
+    data: string;
+    getContextImage(): void;
+}
+
+export default function ContextImage(props: Props): JSX.Element {
+    const {
+        frame, contextImageHide, loaded, data, getContextImage,
+    } = props;
+
+    useEffect(() => {
+        if (!contextImageHide && !loaded) {
+            getContextImage();
+        }
+    }, [frame, contextImageHide, loaded]);
```

Comment:
Do we really need **frame** here? Doesn't **loaded** cover the **frame** functionality?